draft-ietf-rats-corim.md

```diff
@@ -2351,6 +2425,21 @@ For each `rv` entry, the `condition` ECT is compared with an ACS ECT, where the
 
 If the ECTs match except for authority, the `rv` `addition` ECT authority is added to the ACS ECT authority.
 
+#### Processing Domain Membership {#sec-process-dm}
+
+Domain Membership Triples allow an Endorser (for example an Integrator) to issue an authoritative statement about the composition of an Attester as a collection of Environments.
+If the Verifier Appraisal policy requires Domain Membership, then membership triple sets a reference composition of an Attester in a Verifier database.
+At the time of Verification, the reference is then matched with actual composition as reported by an Attester in the Evidence.
+
+Domain Membership Triples are first transformed into an internal representation following the steps mentioned in {{sec-ir-dm-trans}} leading to a representation as specified in {{sec-ir-dm}}.
+
+For each `ECT` entry in the list of `members`, the environment elements (example Class ID) is compared with the equivalent environment elements in ACS Entry with `cmtype` as `evidence`.
```

Comment:
```suggestion
A list of mutable references to matching ACS Entries for a `dm` is constructed in an iterative process over `dm`.`members`.
For every environment in `dm`.`members`, select ECTs from the ACS where `cmtype: &(evidence: 2)` and whose environment matches.
If there are no such ECTs, then the `dm` processing fails.
If there is at least one such ECT, then add all to the list.

At the end of list construction, for each ECT in the list of references, copy the `dm`.`domain-id` to the ECT's `domain-id` field.
If the field is already present and denotes a different domain, then this is an error.
```
I think this could be clearer. I don't think I understood from previous texts that an environment could belong to at most one domain.

Reply:
I have taken your edits to the main comparison section, as I agree the changes `you proposed` are more clear!